dataverse-persistence/src/main/resources/Bundle_en.properties

```diff
@@ -1427,7 +1427,8 @@ file.count={0} to {1} of {2} {2, choice, 0#Files|1#File|2#Files}
 file.count.shown={0} {0, choice, 0#Files Selected|1#File|2#Files}
 file.clearSelection=Clear selection.
 file.numFilesSelected={0} {0, choice, 0#files are|1#file is|2#files are} currently selected.
-file.selectAllFiles=Select all {0} files in this dataset.
+file.selectAllFiles=Select all {0} files in this dataset{1}
+file.selectAllFilesWhileSearching=\u0020matching search term.
```

Comment:
Hmm imo bundle text should be standalone (they don't need another bundle to make sense). To be discussed during daily.